sceptre/nlpsandbox/config/common/nlpsandboxvpc.yaml

```diff
@@ -0,0 +1,12 @@
+template_path: remote/vpc-mini.yaml
+stack_name: nlpsandboxvpc
+stack_tags:
+  Department: "CompOnc"
+  Project: "Nlp Sandbox"
+  OwnerEmail: "thomas.yu@sagebase.org"
+parameters:
+  VpcSubnetPrefix: "10.255.21"
```

Comment:
@zaro0508 said `10.255.31` and @ahayden said `10.255.21`.  Do either of you have a strong preference for either one?

Reply:
either is fine.

Reply:
I'll go with `10.255.21` and update the VPN table accordingly.

Reply:
@zaro0508: Since I'm a beginner with CIDRs, can you confirm that a mini VPC with `10.255.21` corresponds to `10.255.21.0/24`?

Reply:
@tthyer: Thanks for the explanation! I can wrap my mind around the geometric sequence.